asst2/CODE/my_malloc.h

```diff
@@ -2,8 +2,32 @@
 #define ASST1_MY_MALLOC_H
 
 #include <stddef.h>
+#include <string.h>
+#include "my_scheduler.h" 
 
-void* myallocate(size_t size);
-void mydeallocate(void* ptr);
+#define THREADREQ 1
+#define LIBRARYREQ 0
+//#define malloc(x) myallocate(x, __FILE__, __LINE__, THREADREQ)
+//#define free(x) mydeallocate(x, __FILE__, __LINE__, THREADREQ)
+#define MEMSIZE 8388608
+#define SEGMENTSIZE 64
```

Comment:
Can you pull in the global variables for malloc here? They're in `global_vals.h`

Reply:
hmm wait, so move `MEMSIZE` and `SEGMENTSIZE` into global_vals.h? or move the globals from my_malloc.c into my_malloc.h?

Reply:
Yes, can you put these as const values in `global_vals`?